app/presenters/bid_status_flash_factory.rb

```diff
@@ -7,8 +7,10 @@ def initialize(auction:, user:)
   end
 
   def create
-    if available?
+    if available? && auction.type == 'reverse'
```

Comment:
I will admit I don't like it when Presenters are querying the auction.type like this. It feels a bit wrong. Is this somthing we could encapsulate as a method within the respective Rules objects?